<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCSettingAddByScanf</class>
 <widget class="QWidget" name="QCSettingAddByScanf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
background-color: rgb(85, 170, 255);</string>
     </property>
     <property name="text">
      <string>新增质控品</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>质控名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>批号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>生产日期：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>有效期：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CusLineEdit" name="BarCodeArea" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>质控模式：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="QCModelCombox">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="CusLineEdit" name="RackIndex" native="true"/>
     </item>
     <item>
      <widget class="CusLineEdit" name="ElementIndex" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReturnBtn">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CusLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">CusLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
